pkg/service/schema.go

```diff
@@ -0,0 +1,94 @@
+package service
+
+// ParameterSchemas is a set of optional JSONSchemas that describe
+// the expected parameters for creation and update of instances and
+// creation of bindings.
+type ParameterSchemas struct {
+	ServiceInstances *instancesSchema `json:"service_instance,omitempty"`
+	ServiceBindings  *BindingsSchema  `json:"service_binding,omitempty"`
+}
+
+type instancesSchema struct {
+	Create *ProvisioningParametersSchema `json:"create,omitempty"`
+	Update *UpdatingParametersSchema     `json:"update,omitempty"`
```

Comment:
Can we rename the field as `UpdatingParametersSchema`? It seems it's more easily tied back to existing things this way?